cabal-install/src/Distribution/Client/ProjectPlanning.hs

```diff
@@ -1200,6 +1200,7 @@ planPackages verbosity comp platform solver SolverSettings{..}
     -- respective major Cabal version bundled with the respective GHC
     -- release).
     --
+    -- etc.
```

Comment:
It means the list goes on, but only the start is being shown. The trick is, the list start at the bottom and is truncated at this place.

Reply:
It's just usually etc goes last, not first. And should it be updated for GHC 9.4, 9.6?..

Reply:
Usually lists go from the top of the page towards the bottom. This one goes in reverse and abruptly ends at 9.2, which is why I added "etc." in this place. I guess my joke was that good after all. :)

But I don't think we should need to update this line every release, so some other marker is needed, if "etc." doesn't work. Perhaps we should reverse the list and then "etc." would be fine?

Reply:
Apologies for the typos. Edited.

Reply:
I think it doesn't matter what order you use for the list: it can be ascending or descending -- a matter of taste. But having etc. first is just a bit weird. Although, it may read as a joke, I guess. I'm so bad at humor!.. Anyways, over to you. I just noted this weirdness... And no, I don't think it's necessary to update the list with every release -- we have too many places to update already!

Reply:
Yeah, I checked it, the list goes

```
    -- GHC 9.2   needs  Cabal >= 3.6
    -- GHC 9.0   needs  Cabal >= 3.4
    -- GHC 8.10  needs  Cabal >= 3.2
    -- GHC 8.8   needs  Cabal >= 3.0
    -- GHC 8.6   needs  Cabal >= 2.4
    -- GHC 8.4   needs  Cabal >= 2.2
    -- GHC 8.2   needs  Cabal >= 2.0
    -- GHC 8.0   needs  Cabal >= 1.24
    -- GHC 7.10  needs  Cabal >= 1.22
```

So if you put "etc." at the end, you are suggesting there should be GHCs earlier than 7.10 on the list, but there are not. And the list is reversed, because the code is in such an order. And the code is in such an order so taht the "otherwise" case at the logical place.

Reply:
Oh my! I got you!

Reply:
Haha, I manage to explain the joke. Great. :D